health/micro-ui/web/packages/modules/campaign-manager/src/components/HCMMyCampaignRowCard.js

```
@@ -314,8 +314,8 @@ const HCMMyCampaignRowCard = ({ key, rowData, tabData }) => {
           <div className="digit-results-card-heading">{rowData?.campaignName}</div>
           <div className="digit-results-card-tags">
             {showCancelCampaign ? (
-              <div className="digit-tag-container" style={{ margin: "0rem" }}>
-                <Chip text={`${t(`CANCEL_CAMPAIGN`)}`} onClick={handleCancelClick} hideClose={false} />
+              <div className="digit-tag-container" onClick={handleCancelClick} style={{ margin: "0rem" }}>
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)